tests/run-dumb-init-check.sh

```diff
@@ -23,15 +23,17 @@ container="$1"
 
 username="usersetup"
 username_width=${#username}
-expected="1 $username /usr/bin/dumb-init -- /usr/bin/toaster-entry.py"
+required="^1 $username /usr/bin/dumb-init"
 
 # Use bash -c, because otherwise you must specify an absolute path to ps,
 # because $PATH would not be set.
 actual=`docker exec $container bash -c "ps -w -w h -C dumb-init -o pid:1,user:$username_width,args"`
-
-if [ "$expected" != "$actual" ]; then
-    printf "expected dumb-init not found\n"
-    printf "expected:\n%s\n" "$expected"
+# just make sure that dumbinit is running as pid 1.
```

Comment:
I'm going to remove this comment, because it would be more confusing than anything else without the context of how it was previously.